all-products.html

```diff
@@ -107,10 +139,11 @@ <h2>Product name</h2>
 		</div>
 
 		<footer>
-			<p>Footer text will go here.</p>
-			<p>Back to top button will go on the bottom right corner (non sticky)</p>
+			<a href="#top">Back to Top<br>&#129041;</a>

```

Comment:
Footer has been completed for all pages, HTML and CSS have been updated. No further comments will be made on the footer in this Pull request.